x-pack/plugins/maps/public/connected_components/map/mb/draw_control/draw_circle.ts

```diff
@@ -6,9 +6,9 @@
 
 /* eslint-disable @typescript-eslint/consistent-type-definitions */
 
-// @ts-ignore
-import turf from 'turf';
-// @ts-ignore
+// @ts-expect-error
```

Comment:
Not sure why, but these are the only modules without types so they needed `ts-expect-error`